pkgs/unified_analytics/lib/src/analytics.dart

```diff
@@ -333,6 +288,51 @@ abstract class Analytics {
   ///
   /// The snooze period is defined by the [Survey.snoozeForMinutes] field.
   void surveyShown(Survey survey);
+
```

Comment:
@jacob314 continuing our discussion from the Gerrit CL, how do you feel about this change?